client/services/userService.js

```diff
@@ -0,0 +1,217 @@
+/* eslint-disable import/no-import-module-exports */
+/* eslint-disable consistent-return */
+import axios from 'axios'
+import axiosConfig from '../src/utils/config'
```

Comment:
I think you mean axiosConfigObject, since that is the name of your default export.